lua/terrortown/menus/gamemode/base_gamemodemenu.lua

```diff
@@ -10,6 +10,7 @@ CLGAMEMODEMENU.priority = 0
 CLGAMEMODEMENU.icon = nil
 CLGAMEMODEMENU.title = ""
 CLGAMEMODEMENU.description = ""
+CLGAMEMODEMENU.searchBarPlaceholder = nil
```

Comment:
It took me a few minutes to figure out what this does. This has to be renamed for sure. May I suggest something like `CLGAMEMODEMENU.searchBarPlaceholderText` would probably suffice.